api/config/stripeConfig.py

```diff
@@ -0,0 +1,6 @@
+# Sample config for Stripe Payment
+
+STRIPE_PUBLISHABLE_KEY = 'pk_test_6pRNASCoBOKtIshFeQd4XMUh'
+STRIPE_SECRET_KEY = 'sk_test_BQokikJOvBiI2HlWgH4olfQ2'
+amount = ''
```

Comment:
This has to be included in the model, so that admin can change the functionality

Reply:
ok, will add a model :+1: